namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/pic-probation-offender-events-sub-queue.tf

```diff
@@ -0,0 +1,137 @@
+  {
+    "deadLetterTargetArn": "${module.pic_probation_offender_events_dead_letter_queue.sqs_arn}","maxReceiveCount": 5
+  }
+
+EOF
+
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "aws_sqs_queue_policy" "pic_probation_offender_events_queue_policy" {
+  queue_url = module.pic_probation_offender_events_queue.sqs_id
+
+  policy = <<EOF
+  {
+    "Version": "2012-10-17",
+    "Id": "${module.pic_probation_offender_events_queue.sqs_arn}/SQSDefaultPolicy",
+    "Statement":
+      [
+        {
+          "Effect": "Allow",
+          "Principal": {"AWS": "*"},
+          "Resource": "${module.pic_probation_offender_events_queue.sqs_arn}",
+          "Action": "SQS:SendMessage",
+          "Condition":
+                      {
+                        "ArnEquals":
+                          {
+                            "aws:SourceArn": ["${module.probation_offender_events.topic_arn}"]
+                          }
+                        }
+        }
+      ]
+  }
+
+EOF
+
+}
+
+module "pic_probation_offender_events_dead_letter_queue" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=4.8"
+
+  environment-name       = var.environment-name
+  team_name              = var.team_name
+  infrastructure-support = var.infrastructure-support
+  application            = var.application
+  sqs_name               = "pic_probation_offender_events_queue_dl"
+  encrypt_sqs_kms        = "true"
+  namespace              = var.namespace
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "pic_probation_offender_events_queue" {
+  metadata {
+    name      = "pic-probation-offender-events-sqs-instance-output"
+    namespace = "pic-dev"
+  }
+
+  data = {
+    access_key_id     = module.pic_probation_offender_events_queue.access_key_id
+    secret_access_key = module.pic_probation_offender_events_queue.secret_access_key
+    sqs_ptpu_url      = module.pic_probation_offender_events_queue.sqs_id
+    sqs_ptpu_arn      = module.pic_probation_offender_events_queue.sqs_arn
+    sqs_ptpu_name     = module.pic_probation_offender_events_queue.sqs_name
+  }
+}
+
+resource "kubernetes_secret" "pic_probation_offender_events_dead_letter_queue" {
+  metadata {
+    name      = "pic-probation-offender-events-sqs-dl-instance-output"
+    namespace = "pic-dev"
+  }
+  data = {
+    access_key_id     = module.pic_probation_offender_events_dead_letter_queue.access_key_id
+    secret_access_key = module.pic_probation_offender_events_dead_letter_queue.secret_access_key
+    sqs_ptpu_url      = module.pic_probation_offender_events_dead_letter_queue.sqs_id
+    sqs_ptpu_arn      = module.pic_probation_offender_events_dead_letter_queue.sqs_arn
+    sqs_ptpu_name     = module.pic_probation_offender_events_dead_letter_queue.sqs_name
+  }
+}
+
+resource "kubernetes_secret" "pic_probation_sqs_tool_main_queue" {
+  metadata {
+    name      = "pic-probation-sqs-tool-main-queue"
+    namespace = "pic-dev"
```

Comment:
I don't think this namespace is right, it should probably be `court-probation-dev`